pybossa/importers/importer.py

```diff
@@ -167,9 +168,11 @@ def create_tasks(self, task_repo, project, **form_data):
         validator = TaskImportValidator()
         n_answers = project.get_default_n_answers()
         try:
+            task_state, use_file_url = (u'enrich', True) if project.info.get('enrichments') else (u'ongoing', False)
```

Comment:
does this means when proj has enrichment configured, csv import will always create tasks with enrich state?

Reply:
Yes

Reply:
I changed it to only enrich when `state=enrich` in the CSV import.